assets/dev/js/editor/views/add-section/base.js

```diff
@@ -106,6 +107,15 @@ class AddSectionBase extends Marionette.ItemView {
 		];
 	}
 
+	createContainer() {
+		$e.run( 'document/elements/create', {
+			model: {
+				elType: 'container',
+			},
+			container: elementor.getPreviewContainer(),
+		} );
+	}
+
```

Comment:
If it called once, there is no reason for this method to exists...

Reply:
That's purely for readability, I also do that with variables, so the next person who reads this code won't need to investigate what I've done there, i.e.:

```JS
const isContainerActive = elementorCommon.config.experimentalFeatures.container;

...
// Some stuff
...

if ( isContainerActive ) {
    // whatever
}
```

From my point of view, methods are not just for code reusability, that's also (mainly?) to give a human-friendly name to a piece of code.

Reply:
You are right about this, but 
```
$e.run( 'document/elements/create', {
    model: {
        elType: 'container',
    },
    container: elementor.getPreviewContainer(),
} );
```
Is more readable/understand able than this:
```
this.createContainer();
```

Because I don't know which exactly container it may create vs command which is fully understood able.

Reply:
@matipojo what do you think?